lib/disk_help.sh

```diff
@@ -158,7 +158,7 @@ MakeSwapDump() {
 
     # We want at least 10GB left free after swap/dump
     # If we can't make swap/dump at least 1G each, don't bother
-    ((usable = free - 10))
+    usable=`echo "scale=0;$free * 9/10 - 11" | /bin/bc`
```

Comment:
You can just do this in bash rather than calling out to bc.

```
((usable = free * 9 / 10 -  11))
```

Reply:
thanks